Makefile

```diff
@@ -399,6 +399,11 @@ integration-functional-only: out/minikube$(IS_EXE) ## Trigger only functioanl te
 .PHONY: html_report
 html_report: ## Generate HTML  report out of the last ran integration test logs.
 	@go tool test2json -t < "./out/testout_$(COMMIT_SHORT).txt" > "./out/testout_$(COMMIT_SHORT).json"
+	# install gopogh if not already installed
+	@if ! command -v gopogh >/dev/null 2>&1; then \
+		echo "gopogh not found, installing..."; \
+		GOBIN=$(shell go env GOPATH)/bin go install github.com/medyagh/gopogh/cmd/gopogh@latest; \
```

Comment:
Why do we need to set GOBIN? do you want to install gopogh locally only for the project or in the standard location?

Reply:
the install path is 
```/Users/medya/go/bin/gopogh```

some ppl dont have go/bin in their Path (the default) only have GOPATH, it is one way to remove all headaches and jsut give them the HTML_Report